profiles/seccomp/default.json

```diff
@@ -601,7 +601,6 @@
 				"mount",
 				"mount_setattr",
 				"move_mount",
-				"name_to_handle_at",
```

Comment:
@neersighted I noticed we remove this one from the list here, but don't add it to the allow-list, https://github.com/moby/moby/blob/a78c06e3f06e6f55518e1ba2bf9f723cf19d0a65/profiles/seccomp/default_linux.go#L401

Doesn't that mean that it now gets the default (`specs.ActErrno` /  `SCMP_ACT_ERRNO`)?
https://github.com/moby/moby/blob/a78c06e3f06e6f55518e1ba2bf9f723cf19d0a65/profiles/seccomp/default_linux.go#L818